src/sonic-yang-models/yang-models/sonic-device_metadata.yang

```diff
@@ -300,6 +300,18 @@ module sonic-device_metadata {
                     description "Set source of anchor route";
                 }
 
+                leaf orch_dash_zmq_enabled {
+                    type boolean;
+                    description "Enable ZMQ feature on APPL_DB DASH tables.";
```

Comment:
Now that you have a dedicate flag for dash orchs to use zmq, can we deprecate the current '-q' option with zmq_server_address that is currently passed to orchagent in smartswitch to enable zmq b/w gnmi and orchagent and use this instead? Why not have a consistent way to enable zmq b/w orchagent and all other components like gnmi, fpmsyncd etc..?

Ref: https://github.com/sonic-net/sonic-buildimage/pull/16661/

Reply:
For Dash, we plan to migrate to enable/disable with the orch_dash_zmq_enabled feature flag.

But the -q option will keep, because the ZMQ address is generated according to the mgmt_ip and eth0-midplane interface status.

There will be another PR in orchagent to make it enable/disable ZMQ with DECICE_METADATA feature flags.

Reply:
okay, please link all relevant zmq PRs together for convenience when you get a chance.